### PR TITLE
Add svccidr support

### LINKED ISSUE #173: sealer run support svccidr
```
sealer run --svccidr x.x.x.x/x
```
Similar to --interface flag, Reference:
   https://github.com/alibaba/sealer/pull/163/files

### PR DESCRIPTION
**Pull request by PatHoo**
### Describe what this PR does / why we need it
`sealer run --svccidr x.x.x.x/x ...`

### Does this pull request fix one issue?
Fixes #173
sealer run support svccidr

### CODE REVIEW

**Comment by cuisongliu** [comment]
please resolve conflicts @PatHoo

File: sealer/cmd/run.go [hunk 1]
```diff
@@ -54,4 +54,5 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.Pk, "pk", "", cert.GetUserHomeDir()+"/.ssh/id_rsa", "set baremetal server private key")
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.Interface, "interface", "i", "", "set default network interface name")
+	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "", "10.96.0.0/12", "Use alternative range of IP address for service VIPs")
```
Comment (cuisongliu) [inline sealer/cmd/run.go:57]: remove default value,default value should be image  Clusterfile provide

File: apply/run.go [hunk 2]
```diff
@@ -62,11 +72,18 @@ func (c *ClusterArgs) SetClusterArgs() error {
...
 		c.cluster.Spec.SSH.PkPasswd = c.pkPasswd
-		return nil
+		return err
 	}
 	if c.interfaceName != "" {
 		c.cluster.Spec.Network.Interface = c.interfaceName
 	}
+	if c.podCidr != "" && IsCidrString(c.podCidr) {
+		c.cluster.Spec.Network.PodCIDR, err = utils.ParseCIDRString(c.podCidr)
+		return err
```
Comment (fanux) [inline apply/run.go:82]: Should not return here.  if err != nil return err

File: apply/run.go [hunk 3]
```diff
@@ -62,11 +72,18 @@ func (c *ClusterArgs) SetClusterArgs() error {
...
+	}
+	if c.svcCidr != "" {
```
Comment (fanux) [inline apply/run.go:84]: you can check the input value is valid, like podCidr

File: sealer/cmd/run.go [hunk 4]
```diff
@@ -54,4 +54,6 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.Pk, "pk", "", cert.GetUserHomeDir()+"/.ssh/id_rsa", "set baremetal server private key")
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.Interface, "interface", "i", "", "set default network interface name")
+	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '192.168.1.0/24'")
```
Comment (fanux) [inline sealer/cmd/run.go]: set pod CIDR. example ...

File: sealer/cmd/run.go [hunk 5]
```diff
@@ -54,4 +54,6 @@ func init() {
...
+	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "", "", "Use alternative range of IP address for service VIPs")
```
Comment (fanux) [inline sealer/cmd/run.go:57]: set service IP CIDR. example ...

Comment (oldthreefeng) [reply sealer/cmd/run.go:57]: if you dont need a short flag . just use` runCmd.Flags().StringVar()` method.

**Review by cuisongliu — APPROVED** [review]

File: sealer/cmd/run.go (hunk 5 above)
Comment (PatHoo) [reply sealer/cmd/run.go]: done

**Pull request merged**